cloudpebble/settings.py

```diff
@@ -1,12 +1,15 @@
 # encoding: utf-8
 # Django settings for cloudpebble project.
 
+import sys
 import os
 import socket
 import dj_database_url
+
 _environ = os.environ
 
 DEBUG = _environ.get('DEBUG', '') != ''
+TESTING = 'test' in sys.argv
```

Comment:
r.e. is it the right test: Tests only ever get run with `python manage.py test` so basically it works, and it's also the answer that you'll find on StackOverflow.